kotlinpoet/src/commonMain/kotlin/com/squareup/kotlinpoet/CodeWriter.kt

```diff
@@ -161,11 +161,19 @@ internal class CodeWriter constructor(
       emit(KModifier.PUBLIC.keyword)
       emit(" ")
     }
+
+    // public always being an implicit modifier in addition to whatever we inherited.
+    // we don't want to throw away the implicit modifier we inherited
+    val implicitModifierContainsAtLeastTwoAccessMonitors = implicitModifiers.count {
```

Comment:
What are "access monitors"?

Reply:
hey! access modifiers are the set of internal, private, public, protected. based that nomenclature off this comment https://github.com/square/kotlinpoet/blob/1242b415d1fb54aa78e318e3c0694b0637aaee69/kotlinpoet/src/commonMain/kotlin/com/squareup/kotlinpoet/KModifier.kt#L31

Reply:
Ah, I see! The code says "access monitors", which looks like a typo!

Reply:
even if your comment asking what are 'access monitors', my brain still read that as access modifiers 🙃 whoops! fixed the typo

Reply:
Happens to me all the time! No worries & thanks for fixing.